packages/config/index.js

```diff
@@ -2,6 +2,7 @@
 
 export const DEBOUNCE_TIME = 250;
 
+export const SHORT_DAY_MONTH_FORMAT = 'EEEEEE, d.M.';
```

Comment:
It's just my opinion so feel free to keep it as it is 😄 . But personally I like format used on Kiwi.com slightly more.
<img width="263" alt="Screenshot 2019-04-05 at 13 34 33" src="https://user-images.githubusercontent.com/2660330/55625023-de2a9b80-57a7-11e9-80ea-b28ccdb7e206.png">

Reply:
I'm OK with that